cli/tsc.rs

```diff
@@ -1513,7 +1507,7 @@ pub fn pre_process_file(
 
     let text = comment.text.to_string();
     if let Some((kind, specifier)) = parse_ts_reference(text.trim()) {
-      let location = parser.get_span_location(comment.span);
```

Comment:
IMO we should keep `get_span_location` on `ParsedModule` as well as `get_span_comments` - they are pretty trivial helpers, but much more user friendly than `module.source_map.lookup_char_pos(comment.span.lo);`.

WDYT @kitsonk?

Reply:
I thought about that and then didn't do it... I should.